The server will no longer raise a `ValueError` when used in a situation where there is an empty workspace
